test-data/local/externally_sampled_params/observations

```diff
@@ -0,0 +1,6 @@
+GENERAL_OBSERVATION OBS {
+   DATA       = RES;
+   INDEX_LIST = 0,2,4,6,8;
+   RESTART    = 0;
```

Comment:
Can we do without `RESTART` here? It probably needs to go away when time-map is evicted.

Reply:
Not sure. We can perhaps remove it from all places whenever time-map is gone.

Reply:
The alternative is to provide `DATE` instead, which does not make much sense for a polynomial example. So better to keep it like this for the time being, then remove everywhere at once.